### ISSUE TITLE
telescope.lua

### ISSUE THREAD

**Issue by gldtn** [CKolkey/config#2]
I wanted to message you privately, but git doesn't have a private messenger lol.. I was wondering if you can provide some help, I'm trying to duplicate your live_grep_args in your telescope.lua but, `"" -g "*.lua"` gets directly inserted into the prompt when ever I hit `<c-g>`, what could I be missing?

https://github.com/gldtn/dotfiles/blob/main/nvim/lua/plugins/telescope.lua

Appreciate your help, thanks for your time!

**Comment by CKolkey** [comment]
Hey :) Sure thing. It's these two lines:

https://github.com/gldtn/dotfiles/blob/main/nvim/lua/plugins/telescope.lua#L11-L13

`replace_termcodes` and `feedkeys` are a handy way to send keyboard input to vim - in this case to prepopulate the input field.

**Comment by gldtn** [comment]
Awesome, I did get it to work by tweaking it a little bit.. with your code, the prompt was inserting a query: `"" -g "*.lua"`, so I removed the double quotes from the beginning; `-g "*.lua"` and it works. Although it seems I can't get it to not cache the picker, even with `cache_picker = false` set.

So everytime I hit `<c-f>` or `<c-w>`, it inserts `-g "*.lua"` and whatever I had already queried in the prompt.

Thank you so much for responding!
